README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://BogdanPopovych.github.io/layout_calendar/)
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/169878175-0d675b67-443e-4426-a929-5bc754a76d7c.png)
Calendar shouldn't be broken if passed number > 31 or < 28